app/src/main/java/unipd/se18/ocrcamera/OCRActivity.java

```diff
@@ -0,0 +1,38 @@
+package com.example.mattia.fotocamera;
+
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.io.File;
+
+/**
+* @author Fasan Giovanni
+*/
+public class OCRActivity extends AppCompatActivity {
+    //private static final String TAG = "AndroidCameraApi";
+    Bundle extras;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_ocr);
+        OcrManager manager = new OcrManager();
+        //initialize OCR library
+        manager.initAPI();
+        extras = getIntent().getExtras();
+        //get a photo's path from MainActivity
+        String path=extras.getString("PATH_I_NEED");
```

Comment:
Is "PATH_I_NEED" a String you chose? To avoid hardcoding could be better to define a final variable at the beginning of the method.